tests/geo_image_test.py

```diff
@@ -248,16 +248,148 @@ def test_georeference_image(tmp_path):
     geoimage.to_grid2d(new_name="CMYK", transform="CMYK")
 
 
+def create_concentric_array(dims=(15, 15, 3)):
```

Comment:
Is this required here? Can we just make something simpler?

Reply:
Sure, something like meshgrid could make it